manifests/client.pp

```diff
@@ -35,7 +35,10 @@
   Optional[String]        $default_pool_inc,
   Optional[String]        $default_pool_diff,
   Integer                 $port                = 9102,
-  Optional[String]        $listen_address      = $facts['networking']['ip'],
+  Variant[String,Array[String[1]]] $listen_address = [
+    $facts['networking']['ip'],
+    $facts['networking']['ip6'],
```

Comment:
`['0.0.0.0', '::']` might make more sense… Opinion?

Reply:
Maybe the most logical is to default to `[]`: do not output configuration and behave the way bacula developers chose it should work out-of-the-box?

Reply:
if bacula has a proper default builtin, I think using that is a good idea.

Reply:
IMHO, no defaults, ie. required param, is better than `[]`.

If defaults is to listen on any interface, we will have to use some hiera settings per OS to workaround the `::` OS-specific behavior.

Reply:
The default is equivalent to what we had, so let's use the default.